src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1414,6 +1417,25 @@ def aks_browse(cmd, client, resource_group_name, name, disable_browser=False,
         dashboard_pod = str(dashboard_pod).split('/')[-1].strip()
     else:
         raise CLIError("Couldn't find the Kubernetes dashboard pod.")
+
+    # Find the port
```

Comment:
My preference, although a preference, would be to try and separate out some of this code into a helper method (e.g. a find_port() method). Although that code might not be called in multiple places I think it will make it more readable and more testable (when we add tests to more of this code) @weinong what do you think?